SwiftUI_Homework/SwiftUI_Homework/UserBuy.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UserBuy.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 22.01.2023.
+//
+
+import SwiftUI
+
+/// UserBuy
```

Comment:
```suggestion
///
```
Везде описание по-русски, но не здесь